src/DynamicRupture/FrictionLaws/LinearSlipWeakening.h

```diff
@@ -195,65 +117,86 @@ class LinearSlipWeakeningBase : public BaseFrictionLaw {
   /*
    * instantaneous healing option Reset Mu and Slip
    */
-  virtual void instantaneousHealing(unsigned int ltsFace) {
+  void instantaneousHealing(unsigned int ltsFace) {
```

Comment:
Currently not, I think Carsten once mentioned that instantaneous healing is never used anyways, and we do not have a reference for this. This is why, I still have the code in here, so that it is not forgotten.

Reply:
ok!